src/loader.js

```diff
@@ -0,0 +1,15 @@
+import Firefox from './data/firefox.yaml';
```

Comment:
As the list of yml files grows, so will having to maintain this file. You can use something like the following to do it dynamically:

```js
const importer = require.context('./', true, /\.yml/);
const keys = [...new Set([...importer.keys()])];

module.exports = keys.reduce(
  (resolvers, key) => ({ ...resolvers, [key]: importer(key).default }),
  {}
);
```

Reply:
Love this.

Reply:
The [website](http://yaml.org/faq.html) seems to recommend using `yaml` instead of `yml` for the extension.

Reply:
We can perhaps add a `$` symbol to the end of the regex to make sure it doesn't match with something like `yaml.js`:

```js
const importer = require.context('./', true, /\.yaml$/);
```

Reply:
Weird, I don't know any project that uses yaml over yml, like Travis, Taskcluster, etc. That said we could support both with `/\.(yml|yaml)$/`.